e2e-tests/micronaut-latest/src/test/java/com/foo/micronaut/latest/MicronautTestController.java

```diff
@@ -0,0 +1,84 @@
+package com.foo.micronaut.latest;
+
+import org.evomaster.client.java.controller.EmbeddedSutController;
+import org.evomaster.client.java.controller.api.dto.AuthenticationDto;
+import org.evomaster.client.java.controller.api.dto.SutInfoDto;
+import org.evomaster.client.java.controller.problem.ProblemInfo;
+import org.evomaster.client.java.controller.problem.RestProblem;
+
+import java.sql.Connection;
+import java.util.List;
+
+public class MicronautTestController extends EmbeddedSutController {
+
+    private MicronautApplication application;
+
+    public MicronautTestController() { setControllerPort(0); }
+
+    @Override
+    public String startSut() {
+        application = new MicronautApplication();
+        try {
+            application.run();
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
+        }
+
+        return "http://localhost:" + application.getPort();
+    }
+
+    protected int getSutPort() {
+        return application.getPort();
+    }
+
+    @Override
+    public boolean isSutRunning() {
+        if (application == null) {
+            return false;
+        }
+        return application.isRunning();
+    }
+
+    @Override
+    public void stopSut() {
+        if (application != null) {
+            try {
+                application.stop();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    @Override
+    public String getPackagePrefixesToCover() {
+        return "com.foo.micronaut.latest";
+    }
+
+    @Override
+    public void resetStateOfSUT() {}
+
+    @Override
+    public List<AuthenticationDto> getInfoForAuthentication() {
+        return null;
+    }
+
+    @Override
+    public Connection getConnection() {
```

Comment:
this might now work... ie compilation error due to latest changes in `master` branch. pull from master, and update if needed